app/models/patient.rb

```diff
@@ -50,7 +50,7 @@ class Patient
 
   # Contact-related info
   enum :voicemail_preference, [:not_specified, :no, :yes]
-  enum :line, [:DC, :MD, :VA] # See config/initializers/lines.rb. TODO: Env var.
+  enum :line, LINES # See config/initializers/env_vars.rb
```

Comment:
No longer in transit, so better and more complete answer (sorry @lomky !)

`LINES` is set as an app constant in `config/initializers/env_var_constants.rb`, reading from `ENV['DARIA_LINES']` and turning that into an array of symbols (generally, `[:DC, :MD, :VA]`. In turn, we can use `LINES` as a constant throughout the app.

If I'm reading you right, it sounds like there's some wisdom to changing the constant `LINES` to `DARIA_LINES` to match the env var names, and I agree with that - if I am in fact reading you right, let's file that as an issue. Thoughts?